_posts/2015-02-09-create-your-application.markdown

```diff
@@ -64,134 +63,135 @@ For more information about generating a Rails application see the [Getting Start
 
   1. Add GeoBlacklight to your `Gemfile`
 
-    ```ruby
-    # In ./Gemfile
-    gem 'blacklight'
-    gem 'geoblacklight'
-    gem 'jettywrapper'
-    ```
+     ```ruby
+     # In ./Gemfile
+     gem 'blacklight'
+     gem 'geoblacklight'
+     gem 'jettywrapper'
+     ```
 
   1. Install required gems and their dependencies
 
-    ```sh
-    $ bundle install
-    ```
+     ```sh
+     $ bundle install
+     ```
 
   1. Run Blacklight generator (with devise authentication)
 
-    ```sh
-    $ rails g blacklight:install --devise
-    ```
+     ```sh
+     $ rails g blacklight:install --devise
+     ```
 
   1. Run GeoBlacklight generator (with a jetty instance for Solr)
 
-    ```sh
-    $ rails g geoblacklight:install --jettywrapper
-    ```
+     ```sh
+     $ rails g geoblacklight:install --jettywrapper
+     ```
 
   1. Run database migrations
 
-    ```sh
-    $ rake db:migrate
-    ```
+     ```sh
+     $ rake db:migrate
+     ```
+
+
+     Quick tip: All of these tasks (1 - 5) are included as part of template to generate a new GeoBlacklight application. To run that generator just run:
+     {: .flash-notice}
 
-    <div class='flash-notice'>
-      Quick tip: All of these tasks (1 - 5) are included as part of template to generate a new GeoBlacklight application. To run that generator just run:
-      
+     ```sh
+     $ rails new your_app_name -m https://raw.githubusercontent.com/geoblacklight/geoblacklight/master/template.rb
+     ```
```

Comment:
In-browser css change. I added ```word-wrap: break-word;``` to the code element. Maybe this could be inserted into the sass.